spec/index.bs

```diff
@@ -2808,7 +2808,7 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
                                     - |textureLayout|.{{GPUTextureBindingLayout/viewDimension}} is
                                         {{GPUTextureViewDimension/"2d"}}.
                                     - |textureLayout|.{{GPUTextureBindingLayout/type}} is not
-                                        {{GPUTextureType/"depth-float"}}.
+                                        {{GPUTextureSampleType/"depth"}} or {{GPUTextureSampleType/"float"}}.
```

Comment:
I'm just (re-)noticing this particular rule. Shouldn't it be possible to load samples out of a multisampled depth texture? The rule may have been added when we thought "depth-comparison" bindings could be used only for comparison and not for loading.

This can be a followup but let's make sure to file an issue if we land this.

Reply:
I wondered about that too after today's call. I'm happy to remove the `depth` case here.